discord/ext/bridge/core.py

```diff
@@ -369,6 +351,8 @@ def wrap(callback):
 
         return wrap
 
+    # TODO: another group decorator for sub-groups
+
```

Comment:
This TODO should either be fixed in this PR or updated for clarity

Reply:
Yeah, I agree, I'll update it

Reply:
Actually I made it its own issue (#2035) for visibility